pkg/litestorage/jetton.go

```diff
@@ -144,3 +144,7 @@ func (s *LiteStorage) GetJettonMasters(ctx context.Context, limit, offset int) (
 	// TODO: implement
 	return nil, nil
 }
+
+func (s *LiteStorage) GetJettonsHoldersCount(ctx context.Context, accountIDs []tongo.AccountID) (map[tongo.AccountID]int32, error) {
+	return nil, nil
```

Comment:
Does it make sense to return `map[tongooAccountID]int32{}, nil`
In this case, GetJettons won't fail when reading jettonsHolders[master.Address]